packages/hoppscotch-common/src/modules/pwa.ts

```diff
@@ -73,6 +73,7 @@ export default <HoppModule>{
   onRootSetup() {
     onMounted(() => {
       usePwaPrompt()
+      useWhatsNewPrompt()
```

Comment:
We can move this out of the PWA module and instead move it into a separate module ?

The update verification is not exclusive to PWAs so better to isolate that part out.